ci/jjb/scripts/openstack-provision-machine.sh

```diff
@@ -26,7 +26,7 @@ fi
 
 if ! openstack security group rule list "${KEY_NAME}" | grep -q "0.0.0.0/0"; then
     openstack security group rule create --protocol icmp "${KEY_NAME}"
-    for tcp_port in 22 80 443 5000 5646 5647 5671 8000 8140 8443 9090; do
+    for tcp_port in 22 80 443 5000 5646 5647 5671 24817 8140 8443 9090; do
```

Comment:
I guess you shoud keep 8000 as the same sec group may be in use by pulp2?

Reply:
I forgot that we are using pulp-ci for Pulp 2 also.

Reply:
@rochacbruno Does pulp2 use 8000 at all? I don't see it here: https://docs.pulpproject.org/en/2.18/user-guide/installation/index.html#prerequisites